src/components/Lyric/Lyric.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import ReactDOM from 'react-dom';
+import "./Lyric.scss";
+
+const Lyrics = (props) => {
+  return ReactDOM.createPortal(
+    (<div className="modal-overlay">
+    <section className="modal-content">
+     <p className="lyrics">{props.error || props.lyrics}<br/>some text</p>
```

Comment:
`some text` is currently showing up at the bottom of each set of lyrics. Is this a placeholder for something else?

Reply:
oh thank you that was a test for formatting the lyrics, I will remove it